isisd/isis_csm.c

```diff
@@ -143,7 +153,9 @@ struct isis_circuit *isis_csm_state_change(enum isis_circuit_event event,
 		assert(circuit);
 		switch (event) {
 		case ISIS_ENABLE:
-			zlog_warn("circuit already enabled");
+			if (IS_DEBUG_EVENTS)
+				zlog_debug("circuit %s already enabled",
+					   circuit->interface->name);
```

Comment:
`circuit->interface` is NULL in `C_STATE_CONF` state.

Reply:
fixed